Maple2.Server.Game/GameServer.cs

```diff
@@ -23,8 +24,6 @@ public class GameServer : Server<GameSession> {
     private readonly Dictionary<long, GameSession> sessions;
     private readonly ImmutableList<SystemBanner> bannerCache;
     private readonly ConcurrentDictionary<int, PremiumMarketItem> premiumMarketCache;
-    private Dictionary<int, Shop> shopCache;
```

Comment:
**suggestion (performance):** Removal of shop caching might affect performance

The removal of shopCache could potentially impact performance. Consider adding a comment explaining why this cache was removed and how performance is maintained without it.